example.json

```
@@ -16,7 +16,7 @@
         "_small": "assets/example-small.png",
         "alt": ""
     },
-    "_routeToLocation": "",
+    "_routeToLocation": "@course",
```

Remove default link from example.json
```suggestion
    "_routeToLocation": "",
```